lib/govuk_tech_docs/table_of_contents/heading.rb

```diff
@@ -12,8 +12,12 @@ def size
         @element_name.scan(/h(\d)/) && $1 && Integer($1)
       end
 
-      def href
-        @page_url + "#" + @attributes["id"]
+      def href(level)
```

Comment:
In theory, will level map to the type of heading? For example, will a level 1 always be an h1 and vice-versa? Just wondering if, rather than passing level in as an argument, we can use the existing `size` method.

Reply:
Ah yes - that'll be cleaner.